README.md

```diff
@@ -1179,6 +1169,31 @@ log = StarkBank.TaxPayment.Log.get!("1902837198237992") |> IO.inspect
 resource and routes, which are all analogous to the TaxPayment resource. The ones we currently support are:
 - DarfPayment, for DARFs
 
+### Preview payment information before executing the payment
+
+You can preview multiple types of payment to confirm any information before actually paying.
+If the "scheduled" parameter is not informed, today will be assumed as the intended payment date.
+Right now, the "scheduled" parameter only has effect on BrcodePreviews.
+This resource is able to preview the following types of payment:
+"brcode-payment", "boleto-payment", "utility-payment" and "tax-payment"
+
+```elixir
+
+previews = StarkBank.PaymentPreview.create!(
+  [
+    %StarkBank.PaymentPreview{
+      id: "00020126580014br.gov.bcb.pix0136a629532e-7693-4846-852d-1bbff817b5a8520400005303986540510.005802BR5908T'Challa6009Sao Paulo62090505123456304B14A",
+      scheduled: '2020-08-13'
+    },
+    %StarkBank.PaymentPreview{
+      id: "34191.09008 61207.727308 71444.640008 5 81310001234321"
+    }
+  ]
+) |> IO.inspect
+```
+
+**Note**: Instead of using PaymentPreview objects, you can also pass each request element in dictionary format
```

Comment:
Padrão no elixir é esse:

**Note**: Instead of using PaymentPreview structs, you can also pass each payment request element in map format